functions.php

```diff
@@ -38,8 +38,48 @@ function get_sql_result($connect, $sql_query, array $data) {
     if ($data) {
 			mysqli_stmt_bind_param($stmt, "s", ...(array)$data);
```

Comment:
```suggestion
			mysqli_stmt_bind_param($stmt, str_repeat('s', count($data)), ...$data);
```

С помощью `str_repeat('s', count($data))` будет сформирована строка ssss по количеству элементов в $data

Reply:
:tada:

Reply:
Да, ужасная функция mysqli_stmt_bind_param, очень сложная сигнатура у неё. Проще было бы для каждого параметра по 1 вызову функции сделать, чтобы отдельно указывать тип и значение.